hw02_unpack_string/unpack.go

```diff
@@ -2,11 +2,74 @@ package hw02unpackstring
 
 import (
 	"errors"
+	"strconv"
+	"strings"
+	"unicode/utf8"
 )
 
 var ErrInvalidString = errors.New("invalid string")
 
-func Unpack(_ string) (string, error) {
-	// Place your code here.
-	return "", nil
+func isDigit(str string) bool {
+	if string('0') <= str && str <= string('9') {
+		return true
+	}
+	return false
+}
```

Comment:
```suggestion
func isDigit(str string) bool {
	return '0' <= str && str <= string'9'
}
```
так тоже будет работать